templates/support/OAuthAccessTokenManager.swift.hbs

```diff
@@ -0,0 +1,126 @@
+//  
+//  {{>frag/generatedBy}}
+//
+
+/** A token manager that will group refresh attempts to prevent concurrent token requests */
+import Foundation
+
+actor OAuthAccessTokenManager {
```

Comment:
I wonder how much of this should be `public` so we can make use of it outside of the API module... I feel like quite a bit of it should be? We might want to interact with the access token etc from app code.

Reply:
We don't need the actor to be public for that. I have exposed the access token on the security client via an async function